components/klaudsolcms/AppModal.js

```diff
@@ -18,15 +18,15 @@ const AppModal = ({
    >
```

Comment:
Good job on this. But I have a question, whenever you click on the outer part of the modal, will the modal close or is it disabled as well? If i'm not mistaken, you'll have to add this to your code too:

```suggestion
  <Modal 
   show={show} 
   onHide={!isLoading ? onClose : null} 
   centered size="lg"
   >
```
Let me know if this works. Thanks!